aws-lc-sys/builder/main.rs

```diff
@@ -178,6 +178,18 @@ fn prepare_cmake_build(manifest_dir: &PathBuf, build_prefix: Option<&str>) -> cm
     cmake_cfg.define("DISABLE_PERL", "ON");
     cmake_cfg.define("DISABLE_GO", "ON");
 
+    if target_vendor() == "apple" {
+        if target_os().trim() == "ios" {
+            cmake_cfg.define("CMAKE_SYSTEM_NAME", "iOS");
+            if target().trim().ends_with("-ios-sim") {
+                cmake_cfg.define("CMAKE_OSX_SYSROOT", "iphonesimulator");
+            }
+        }
+        if target_arch().trim() == "aarch64" {
+            cmake_cfg.define("CMAKE_OSX_ARCHITECTURES", "arm64");
```

Comment:
Is the default x86_64 if this isn't set? Just curious if there could be differences in behavior, depending on the platform we cross-compiling from.

Reply:
It must not be needed. We're not setting it currently and I have no trouble build on a x86 Mac.